### PR TITLE
Feature - integrate ddc pallet to pos network node V2

### PR DESCRIPTION
**Pull request by huythanhcere**
Import DDC-pallet to Validation Node repo via Cargo. [Ticket](https://www.notion.so/cere/Import-DDC-pallet-to-Validation-Node-repo-via-Cargo-df74f781ae684c7398830135845ff8c1)

Update with node V2.0.1.
Add instruction readme file.

### CODE REVIEW

File: .github/workflows/build-and-push-docker-image.yml [hunk 1]
```diff
@@ -3,15 +3,18 @@ on:
   push:
     branches:
       - 'dev-cere'
+      - 'feature/import-ddc-pallet'
```
Comment (AndreiNavoichyk) [inline .github/workflows/build-and-push-docker-image.yml]: Remove before merge to `dev-cere`.

File: bin/node/runtime/src/lib.rs [hunk 2]
```diff
@@ -66,6 +66,7 @@ use pallet_contracts_rpc_runtime_api::ContractExecResult;
 use pallet_session::{historical as pallet_session_historical};
 use sp_inherents::{InherentData, CheckInherentsResult};
 use static_assertions::const_assert;
+pub use pallet_cere_ddc;
```
Comment (AndreiNavoichyk) [inline bin/node/runtime/src/lib.rs:69]: Why it's `pub`?

File: .gitmodules [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[submodule "frame/ddc-pallet"]
```
Comment (AndreiNavoichyk) [inline .gitmodules:1]: Switch to the `dev` branch after DDC-pallet PR will be merged.

File: bin/node/runtime/src/lib.rs (hunk 2 above)
Comment (huythanhcere) [reply bin/node/runtime/src/lib.rs:69]: For other module can call. Like "pub use pallet_transaction_payment::{Multiplier, TargetedFeeAdjustment};"

File: .gitmodules (hunk 3 above)
Comment (huythanhcere) [reply .gitmodules:1]: This specify the target path. I will not change this. But i will add parameter to specify the dev branch in the next commit after you merge the PR in ddc-pallet repo ([url](https://github.com/Cerebellum-Network/ddc-pallet/pull/1))

File: .github/workflows/build-and-push-docker-image.yml (hunk 1 above)
Comment (huythanhcere) [reply .github/workflows/build-and-push-docker-image.yml]: Fixed in local branch. I will push the next commit after you merge the PR in ddc-pallet repo ([url](https://github.com/Cerebellum-Network/ddc-pallet/pull/1))

**Review by AndreiNavoichyk — APPROVED** [review]

**Review by AndreiNavoichyk — APPROVED** [review]
@huythanhcere 
Update the submodule to `b622b9` commit. This is the last commit in ddc repo.

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**